design/one-pager-pluggable-secret-stores.md

```diff
@@ -155,46 +148,40 @@ message DeleteOptions {
 A _Secret Store Plugin_:
 
 - Will be a binary packaged as a container image.
-- Should run a gRPC server listening at Unix domain socket.
-- May support one or more Secret Stores.
+- Will be a k8s deployment behind a service.
```

Comment:
Does it have to be a deployment behind a service? Could it just be any gRPC endpoint?

To be clear I think using deployments makes sense in practice, but I would not artificially limit Crossplane to _only_ be able to communicate with a secret store plugin via a Kubernetes service.

Reply:
removed the limitation